documents/operating.md

```diff
@@ -11,6 +12,17 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* The "**Board of Directors**" ("the **Board**") of the Company, as defined by Pennsylvania state law, shall consist of all current Corporate Overlords of the Company.
+* "**Consensus**" shall mean TKTKTK.
+* "**Quorum**" shall mean TKTK
```

Comment:
A few notes:

1. We should also have a process for updating what Quorum is, as the company changes and grows.  We should define this process later in the document and mention that rules for changing this number are described later in the document

2.  Do we need a quorum at all?  We are unique in that we are setting up mechanisms to "vote" on things asynchronously.  That said, the section on voting hasn't been fleshed out (sorry) -- what I think would be reasonable is that voting can happen over time, but votes are not finalized until a plotting session happens.  (We can have preliminary votes mean something, e.g. if 4 people approve of an idea the Beastmaster has the green light to temporarily approve it / onboard it)

So I just talked myself into saying "yeah, we should have a concept of quorum and plotting sessions should matter."

3. Quorum should be large enough that we can deliberate and cast votes meaningfully, but small enough that we will be able to reasonably expect that level of participation.  I had pitched 70%.  I think that so long as people take their role as CO seriously and are willing to schedule around a 2 hour meeting once a month, we should be able to hit that.

70% of 12 COs would be 9 people.

Thoughts?